docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,79 @@ To investigate further, you can view the details of the build captured as labels
 .Contextual attributes of a Jenkins pipeline execution in Elastic Observability
 image::images/jenkins-pipeline-context.png[Attributes of a Jenkins pipeline execution]
 
+[discrete]
+[[ci-cd-store-jenkins-logs]]
+== Store Jenkins pipeline logs in Elastic
+
+Jenkins pipeline build logs can be sent through OpenTelemetry Protocol (OTLP) to be stored
+in an observability backend alongside the traces of the pipeline builds and Jenkins health metrics.
+Storing logs in an observability backend provides several benefits including:
+
+* Better observability, monitoring, alerting, and troubleshooting of the Jenkins instance
+due to the unification of all the signals in the observability backend.
+* Better traceability and control of long term storage for builds so you can more easily
+audit the Software Delivery Lifecycle.
+* Better scalability and reliability of Jenkins by greatly reducing the quantity of data stored in
+Jenkins and limiting the well known file system performance of Jenkins when storing a large history of builds.
+
+[discrete]
+[[ci-cd-store-jenkins-logs-requirements-configuration]]
+=== Requirements and configuration
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic Observability version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry integrationmust be
```

Comment:
```suggestion
* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry Plugin must be

```

Reply:
We should also update [the open source doc](https://github.com/jenkinsci/opentelemetry-plugin/blob/master/docs/build-logs.md#storing-jenkins-pipeline-logs-in-elastic-elastic-v81) to use "Plugin" instead of "integration".

Reply:
done thanks